examples/wistia/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "wistia",
+  "version": "1.0.0",
+  "description": "Vistia  Widget for contentful",
+  "main": "./src/app.js",
+  "scripts": {
+    "env": "env",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "widget:create": "./node_modules/.bin/better-npm-run widget:create",
+    "widget:update": "./node_modules/.bin/better-npm-run widget:update",
+    "widget:dev": "./node_modules/.bin/better-npm-run widget:dev",
+    "widget:serve": "./node_modules/.bin/better-npm-run widget:serve"
+  },
+  "betterScripts": {
+    "widget:create": {
+      "command": "export CONTENTFUL_MANAGEMENT_ACCESS_TOKEN=$CONTENTFUL_MANAGEMENT_ACCESS_TOKEN && contentful-widget create --space-id $SPACE_ID"
+    },
+    "widget:update": {
+      "command": "export CONTENTFUL_MANAGEMENT_ACCESS_TOKEN=$CONTENTFUL_MANAGEMENT_ACCESS_TOKEN && contentful-widget update --space-id $SPACE_ID --force"
+    },
+    "widget:dev": {
+      "command": "export CONTENTFUL_MANAGEMENT_ACCESS_TOKEN=$CONTENTFUL_MANAGEMENT_ACCESS_TOKEN && contentful-widget update --space-id $SPACE_ID --force --src \"http://localhost:3030/index.html\""
+    },
+    "widget:serve": {
+      "command": "webpack-dev-server --open --inline --colors --progress --display-error-details --display-cached --port $PORT --content-base ./dist"
+    }
+  },
+  "author": "frederik@contentful.com",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-core": "^6.7.2",
+    "babel-loader": "^6.2.4",
+    "babel-preset-es2015": "^6.6.0",
+    "better-npm-run": "0.0.8",
+    "contentful-widget-cli": "^1.1.3",
+    "contentful-widget-sdk": "^0.1.1",
+    "css-loader": "^0.23.1",
+    "ejs-loader": "^0.2.1",
+    "html-webpack-plugin": "^2.10.0",
+    "lodash": "^4.6.1",
+    "style-loader": "^0.13.1",
+    "webpack": "^1.12.14",
+    "webpack-dev-server": "^1.14.1"
```

Comment:
Please use `"webpack-dev-server": "1.14.0"`. This [bug](https://github.com/webpack/webpack-dev-server/issues/381) makes `1.14.1` unusable :(